vehicle/toyota/identity_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build integration
```

Comment:
This will now always fatal the test?

Reply:
Ist schon gefixt. Habe ich aus Versehen mit committed.

So werden die Tests wieder nur ausgeführt, wenn das integration tag gesetzt ist:
go test -tags=integration ./...